layers/meta-balena-allwinner/recipes-bsp/u-boot/u-boot_2019.04.bbappend

```diff
@@ -3,17 +3,15 @@ inherit resin-u-boot pythonnative
 
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
-#remove the resin-specific-env-integration-kconfig.patch patch from meta-resin
-#and the 0001-nanopi_neo_air_defconfig-Enable-eMMC-support.patch patch from
-#meta-sunxi because these fail to apply
-SRC_URI_remove = "file://0001-nanopi_neo_air_defconfig-Enable-eMMC-support.patch \
-                  file://resin-specific-env-integration-kconfig.patch"
+# remove the resin-specific-env-integration-kconfig.patch patch from meta-resin
```

Comment:
`The eMMC patch has been removed because it applies
from meta-balena.` I think this is a typo, meta-balena should not and I think it does not add this patch. Please redo the commit log

Reply:
Yes, It's a typo. Fixed it.